py3status/modules/xrandr.py

```diff
@@ -29,6 +29,9 @@
         (default None)
     format: display format for xrandr
         (default '{output}')
+    hide_if_single_combination: hide if only one combination is available,
+        has effect only if 'format' uses square brackets, like: '[{output}]'
```

Comment:
I'd remove this `L33` explaination and just update `L31` and `L101` with `[{output}]`.
We generally don't want to pass `''` if the default format does not have `[]` in there.